deployments/liqo/templates/webhooks/job-patch/job-create-secret.yaml

```
@@ -1,4 +1,5 @@
-{{- $cfg := (merge (dict "name" "webhook-certificate-patch" "module" "webhook-certificate-patch") .) -}}
+{{- $cfg := (merge (dict "name" "webhook-certificate-patch-pre" "module" "webhook-certificate-patch") .) -}}
+{{- $rbacCfg := (merge (dict "name" "webhook-certificate-patch") .) -}}
```

```suggestion
{{- $rbacConfig := (merge (dict "name" "webhook-certificate-patch") .) -}}
```

Nit. Looks more consistent to me with the other variables below (same for the other file)